sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/CHANGELOG.md

```diff
@@ -15,6 +15,9 @@
   `APPLICATIONINSIGHTS_CONNECTION_STRING` is present in `IConfiguration`.
   ([#45292](https://github.com/Azure/azure-sdk-for-net/pull/45292))
 
+* Fixed a bug in LiveMetrics that counted all manually created Dependencies as failures.
```

Comment:
Is this the only issue that was fixed?

Reply:
No, I need to add another entry about updating all Dependencies to match the Exporter's mappings.